pwiz_tools/Skyline/Documentation/Tutorials/DIA-PASEF/en/index.html

```diff
@@ -673,7 +673,7 @@ <h1>Quantitative comparison </h1>
         To perform a simple pairwise group comparison inside Skyline do the following:
     </p>
     <ul>
-        <li>On the <b>View</b> menu, choose <b>Other Grids</b>, then <b>Group Comparison</b>, and click <b>Add</b>.</li>
+        <li>On the <b>View</b> menu, choose <b>Live Reports</b>, then <b>Group Comparisons</b>, and click <b>Add...</b>.</li>
```

Comment:
We don't include the ... Changing that now everywhere would be much more work, especially since Chinese use a different ...